blenderproc/python/renderer/SegMapRendererUtility.py

```diff
@@ -300,13 +300,18 @@ def _set_world_background_color(color: mathutils.Vector):
     nodes = bpy.context.scene.world.node_tree.nodes
     links = bpy.context.scene.world.node_tree.links
 
+    background_node = Utility.get_the_one_node_with_type(nodes, "Background")
+
     # Unlink any incoming link that would overwrite the default value
-    if len(nodes.get("Background").inputs['Color'].links) > 0:
-        links.remove(nodes.get("Background").inputs['Color'].links[0])
+    if len(background_node.inputs['Color'].links) > 0:
+        links.remove(background_node.inputs['Color'].links[0])
     # Set strength to 1 as it would act as a multiplier
-    nodes.get("Background").inputs['Strength'].default_value = 1
-    nodes.get("Background").inputs['Color'].default_value = color + [1]
+    background_node.inputs['Strength'].default_value = 1
+    background_node.inputs['Color'].default_value = color + [1]
```

Comment:
Shouldn't we add a check here, to make sure that color truely has only 3 elements before appending one.